Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM    node:alpine
+
+RUN     mkdir /app
+WORKDIR /app
+
+ADD     . .
```

Comment:
When we add everything, aren't we copying `node_modules` too? Wouldn't it be enough to copy `package.json` and the `src` folder?